decentraland/unity-renderer
# feat: moving player prefs to local storage

## What does this PR change?

Fixes #4137 

Since PlayerPrefs were not being reliabily stored in IndexedDB, me are moving them to LocalStorage.

## How to test the changes?

1. Go to https://play.decentraland.zone/?explorer-branch=test/moving-players-prefs-to-local-storage
2. Play around with your settings. Open the Settings, change them, close the Explore menu so they get saved. Refresh the website, and check that they have been saved correctly.
3. Go through the test cases enumerate in the issue and check that they are all solved
4. Change the settings to Ultra. Check that they remain in Ultra and dont change to custom

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
